pkg/agent/sysadvisor/plugin/qosaware/resource/cpu/region/provisionpolicy/policy_rama.go

```diff
@@ -144,3 +172,36 @@ func (p *PolicyRama) sanityCheck() error {
 
 	return errors.NewAggregate(errList)
 }
+
+func (p *PolicyRama) getReclaimStatus() (usage float64, cnt int) {
+	usage = 0
+	cnt = 0
+
+	f := func(podUID string, containerName string, ci *types.ContainerInfo) bool {
+		if ci.QoSLevel != apiconsts.PodAnnotationQoSLevelReclaimedCores {
+			return true
+		}
+
+		containerUsage, err := p.metaServer.GetContainerMetric(podUID, containerName, consts.MetricCPUUsageContainer)
+		if err != nil {
+			containerUsage = ci.CPURequest
+		}
+
+		// FIXME: metric server doesn't support to report cpu usage in numa granularity,
+		// so we split cpu usage evenly across the binding numas of container.
+		if p.bindingNumas.Size() > 0 {
```

Comment:
we assume bindingNumas.size() should always = 1 (if it is larger than 0), right ? like the dedicated region assumes there should contain only one dedicated container, we should change the formula should be

usgae * cpu size / numa size * p.bindingNumas.size() ?

Reply:
We want to calculate the cpu usage intersected with this region of a reclaimed container, which should be: usage * cpus intersected with this region / all cpus in container's topology assignment.